src/components/Thanks/ThanksPageDonationOnly.vue

```diff
@@ -0,0 +1,387 @@
+<template>
+	<div>
+		<generic-promo-banner
+			v-show="monthlyDonationAmount"
+			:promo-banner-content="promoBannerContent"
+		/>
+		<div class="row page-content">
+			<div class="large-2"></div>
+			<div class="small-12 large-8 columns thanks">
+				<div class="hide-for-print">
+					<h1 class="tw-text-h1 tw-text-center tw-mb-2">
+						{{ headerMsg }}
+					</h1>
+					<p class="tw-text-center tw-mb-2">
+						As a donor, you help us bring more loans to people in need. Watch Manal’s story:
+					</p>
+					<div class="tw-my-4">
+						<iframe
+							class="tw-aspect-video tw-mx-auto tw-rounded tw-w-full tw--mb-1.5 md:tw--mb-1"
+							width="560"
+							height="315"
+							:src="`https://www.youtube.com/embed/${youtubeId}?rel=0`"
+							title="YouTube video player"
+							frameborder="0"
+							allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; web-share"
+							allowfullscreen
+						></iframe>
+					</div>
+					<p class="tw-my-2">
+						How a Kiva loan helped Manal grow her business in Palestine
+					</p>
+					<p class="tw-mb-2">
+						As an entrepreneur, Manal faces the usual challenges - finding customers, stocking up on
+						materials, and keeping up with the competition. But as a woman, Manal has also had to shut
+						down those in her community that disapprove of her running a business.<br><br>
+						When she opened her own sewing workshop five years ago in a village just south of Jenin,
+						Palestine, Manal was met with some tart criticism about her work.<br><br>
+						“It was the competitiveness, which was expected,” recalls the 49 year-old mother of seven.
+						“Belittling my business, most people would say ‘how is your business even supposed to succeed
+						across this town?”<br><br>
+						With the help of a Kiva loan, Manal has not only succeeded but expanded—showing the naysayers
+						that she has a firm handle on her ventures and her livelihood.
+					</p>
+					<h3	class="tw-mt-1 tw-mb-3 tw-text-left">
+						Share this story and your impact
+					</h3>
+					<div class="row tw-mt-3">
+						<div class="large-2"></div>
+						<div class="small-12 large-8 columns">
+							<div class="share__social social">
+								<a
```

Comment:
Please refactor all these share buttons to be like the ones in: src/components/Thanks/ThanksPageCommentAndShare.vue -- note the use of the `showSharePopUp` function